sql/parser/builtins.go

```diff
@@ -331,12 +334,23 @@ var builtins = map[string][]builtin{
 			types:      typeList{},
 			returnType: DummyFloat,
 			impure:     true,
-			fn: func(args DTuple) (Datum, error) {
+			fn: func(_ EvalContext, args DTuple) (Datum, error) {
 				return DFloat(rand.Float64()), nil
 			},
 		},
 	},
 
+	"unique_id": {
```

Comment:
Naming thoughts: `uuid_k_sorted`, `uuid_ks`, `k_sorted_unique_id`. For the purely random ID we could name the function `uuid_v4`. Preferences?